docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,69 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement changes various auxiliary properties of a view.
+
+
+#### Rename view
+Rename the existing view. If the view name already exists in the database, an exception is thrown. This operation does 
+support moving the views cross databases. 
+##### Syntax
+{% highlight sql %}
+ALTER VIEW [db_name.]view_name RENAME TO [db_name.]new_view_name
+{% endhighlight %}
+
+
+#### Set view properties
+Set one or more properties of an existing view. The properties are the key value pairs. If the properties' keys exist, 
+the values are replaced with the new values. If the properties' keys does not exist, the key value pairs are added into 
+the properties.
+##### Syntax
+{% highlight sql %}
+ALTER VIEW view_name SET TBLPROPERTIES (key1=val1, key2=val2, ...)
```

Comment:
Ony a view_name is allowed here ? Can't this be a identifier which may be qualified ?

Reply:
sure, will change